scarlet/cms/templates/cms/insert_media.html

```diff
@@ -3,11 +3,13 @@
 {% block page_class %}insert-media{% endblock page_class %}
 
 {% block popup_header %}
+<!--
```

Comment:
Normally I prefer to delete code and rely on version control but since we expect these to eventually come back can you add a comment here that says "TODO: Uncomment tabs and fix issues with inserting Audio and Video"

Reply:
Comment added :+1: